packages/pac-proxy-agent/src/index.ts

```diff
@@ -103,7 +103,7 @@ export class PacProxyAgent<Uri extends string> extends Agent {
 	 *
 	 * @api private
 	 */
-	private getResolver(): Promise<FindProxyForURL> {
+	private async getResolver(): Promise<FindProxyForURL> {
```

Comment:
This still needs to be an `async` method

Reply:
done